VespaGAS/ViewModels/GasCalculatorViewModel.swift

```diff
@@ -1,5 +1,36 @@
 import Foundation
+import RxSwift
 
-public struct GasCalculatorViewModel {
+struct GasCalculatorViewModel {
+    private let disposeBag = DisposeBag()
+    private let oilValueSubject = BehaviorSubject(value: "0%")
```

Comment:
Good idea, I have refactored the ViewModel with your proposal and now it looks way better! 👍 
Just a consideration about memory management, am I doing it right? Do I need to dispose something in this ViewModel?